localisation/000_Other_title_names.csv

```diff
@@ -380,3 +380,25 @@ k_quelthalas;Quel'Thalas;Quel'Thalas;Quel'Thalas;;Quel'Thalas;;;;;;;;;x
 k_quelthalas_adj;Thalassian;;thalassisch;;;;;;;;;;;x
 b_jorg_townsend;Jorg;;;;;;;;;;;;;x
 b_jorg_townsend_adj;Jorg;;;;;;;;;;;;;x
+k_illidari;Illidari;;;;;;;;;;;;;x
+k_illidari_adj;Illidari;;;;;;;;;;;;;x
+k_blasted_lands;Blasted Lands;;;;;;;;;;;;;x
+k_blasted_lands_adj;Blasted Lands;;;;;;;;;;;;;
+d_southsea_freebooters;Southsea Freebooters;;S�dsee Freibeuter;;;;;;;;;;;x
+d_southsea_freebooters_adj;Southsea Freebooter;;freibeuterisch;;;;;;;;;;;x
```

Comment:
Should we shorten this to Freebooter to match the translation?

Reply:
Done in a93119cbd61bf6073101068c5739f11852efab68.